examples/Atlas/+atlasControllers/AtlasPlanEval.m

```diff
@@ -2,7 +2,7 @@
 % A PlanEval which includes some Atlas-specific tweaks. Specifically, it
 % provides a cache of useful robot properties which may be expensive to look
 % up at runtime.
-  properties (Access = private)
+  properties (Access = protected)
```

Comment:
I did this to make sure that they weren't being modified outside this class... Are you changing robot property cache? If it's being modified on the fly I can't get rid of it as an argument of the getQPControllerInput method.